CHANGELOG.md

```diff
@@ -23,6 +23,9 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
   [Samuel Giddins](https://github.com/segiddins)
   [#4124](https://github.com/CocoaPods/CocoaPods/issues/4124)
 
+* Support for adding dependency target vendored libraries and frameworks to build settings.
```

Comment:
* [ ] Add two spaces at the end of the description to force a line-mark.
* [ ] Add a link to this PR for reference below your name.